BrainPortal/app/controllers/data_providers_controller.rb

```diff
@@ -89,7 +89,7 @@ def new #:nodoc:
   def create #:nodoc:
     @provider = DataProvider.sti_new(data_provider_params)
     @provider.user_id  ||= current_user.id # disabled field in form DOES NOT send value!
-    @provider.group_id ||= (( current_project && current_project.id ) || current_user.own_group.id)
+    @provider.group_id ||default_target_group_id
```

Comment:
This is wrong; this used to be an assign statement `||=` and you changed it to a NOOP `||`

Reply:
corrected